packages/app-cli/app/command-mkbook.test.ts

```diff
@@ -0,0 +1,50 @@
+import { setupDatabaseAndSynchronizer, switchClient } from '@joplin/lib/testing/test-utils';
+import { setupCommandForTesting, setupApplication } from './utils/testUtils';
+import Folder from '@joplin/lib/models/Folder';
+const Command = require('./command-mkbook');
+
+
+describe('command-mkbook', function() {
```

Comment:
There was a recent eslint change that causes this to fail. If you could change it to `() =>` we're good to merge!

Reply:
Done!